.env.dev

```diff
@@ -36,3 +36,5 @@ REACT_APP_KS_SETTING_API=https://ks-setting.dev.kyberengineering.io/api
 REACT_APP_GTM_ID=
 
 REACT_APP_GOOGLE_RECAPTCHA_KEY=6Lc7yg0iAAAAALhS4C4Ez5hqMBub8hKdFvvr9sfc
+
+REACT_APP_POOL_FARM_BASE_URL=https://pool-farm.kyberswap.com
```

Comment:
missed update this for .env.stg

Reply:
They don't have API for staging. Only prod

Reply:
Good catch. Thanks